management/src/main/java/org/ehcache/management/registry/DefaultCollectorService.java

```diff
@@ -84,7 +85,9 @@ public synchronized void start(ServiceProvider serviceProvider) {
     cacheManager = serviceProvider.getService(CacheManagerProviderService.class).getCacheManager();
     scheduledExecutorService = serviceProvider.getService(ExecutionService.class).getScheduledExecutor(configuration.getCollectorExecutorAlias());
 
-    cacheManager.registerListener(this);
+    if (cacheManager instanceof ObservableCacheManager) {
```

Comment:
If this test evaluates to false, this is going to make the cache manager silently un-monitorable. There should at least be a warning about that.